build.jam

```diff
@@ -0,0 +1,34 @@
+# Copyright René Ferdinand Rivera Morell 2023
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+project /boost/filesystem
+    : common-requirements
+        <library>/boost/assert//boost_assert
```

Comment:
As for Boost.Atomic, shouldn't private and public dependencies be separate and specified for the library instead of the project?